GVFS/FastFetch/FastFetchVerb.cs

```diff
@@ -54,6 +54,15 @@ public class FastFetchVerb
             HelpText = "Checkout the target commit into the working directory after fetching")]
         public bool Checkout { get; set; }
 
+        [Option(
+            "force-checkout",
+            Required = false,
+            Default = false,
+            HelpText = "Force FastFetch to fetch and checkout content as if the current repo had just been initialized." +

```

Comment:
I don't think this flag actually affects what we "fetch", at least not directly. It does affect what files we want to checkout, but if we already have the blobs, we're not going to re-download them.